packages/mobile/src/store/auth/selectors.ts

```diff
@@ -2,16 +2,23 @@ import { RootState } from '~/common/types/types';
 
 const selectCurrentUser = (state: RootState) => state.auth.user;
 
+const selectUserId = (state: RootState) => state.auth.user?.id;
+
 const selectUserPhone = (state: RootState) => state.auth.user?.phone;
 
 const selectPhoneVerified = (state: RootState) =>
   state.auth.user?.phoneVerified;
```

Comment:
Such precise selectors doesn't make much sense. We can replace it with single `selectCurrentUser`

Reply:
corrected